test/unionFindWithWeight.spec.js

```diff
@@ -3,91 +3,101 @@
 const algolib = require('../index.js');
 
 describe.only('#unionFindWithWeight', function() {
```

Comment:
do you have a test that checks that a smaller component will conect to a bigger one?